schematic/store/synapse.py

```diff
@@ -85,9 +85,9 @@ def __init__(
             raise MissingConfigValueError(("synapse", "master_fileview"))
 
     @staticmethod
-    def login(token=None, access_token=None):
+    def login(token=None, access_token=None, input_token=None):
```

Comment:
@linglp we can proceed with these arguments and logic for now here. However, could you check with Anthony and Rong if input_token coincides with either access_token or token parameter. I should have flagged that earlier, but just noticed we may be repeating logic.

Reply:
Thanks Milen! I have double checked with Anthony and the input_token works!